test/integration/targets/nxos_aaa_server_host/tests/common/radius.yaml

```diff
@@ -0,0 +1,128 @@
+---
+- debug: msg="START TRANSPORT:{{ connection.transport }} nxos_aaa_server_host radius.yaml sanity test"
+
+- name: "Setup"
+  nxos_aaa_server_host: &remove
+    server_type: radius
+    address: 8.8.8.8
+    state: absent
+    provider: "{{ connection }}"
+  ignore_errors: yes
+
+- block:
+  - name: "Configure radius server defaults"
+    nxos_aaa_server_host: &configure_default_radius
+      server_type: radius
+      address: 8.8.8.8
+      state: present
+      provider: "{{ connection }}"
+    register: result
+
+  - assert: &true
+      that:
+        - "result.changed == true"
+
+  - name: "Check Idempotence"
+    nxos_aaa_server_host: *configure_default_radius
+    register: result
+
+  - assert: &false
+      that:
+        - "result.changed == false"
+
+  - name: "Remove radius server configuration"
+    nxos_aaa_server_host: *remove
+    register: result
+
+  - assert: *true
+
+  - name: "Check Idempotence"
+    nxos_aaa_server_host: *remove
+    register: result
+
+  - assert: *false
+  
+  - name: "Configure radius server non defaults"
+    nxos_aaa_server_host: &configure_radius_non_default
+      server_type: radius
+      address: 8.8.8.8
+      host_timeout: 25
+      auth_port: 2083
+      acct_port: 2084
+      state: present
+      provider: "{{ connection }}"
+    register: result
+  
+  - assert: *true
+
+  - name: "Check Idempotence"
+    nxos_aaa_server_host: *configure_radius_non_default
+    register: result
+
+  - assert: *false
+
+  - name: "Remove radius server configuration"
+    nxos_aaa_server_host: *remove
+    register: result
+
+  - assert: *true
+
+  - name: "Configure radius server with clear text pwd"
+    nxos_aaa_server_host: &configure_radius_clear_text
+      server_type: radius
+      address: 8.8.8.8
+      host_timeout: 25
+      auth_port: 2083
+      acct_port: 2084
+      encrypt_type: 0
```

Comment:
@rahushen after testing `nxos_aaa_server_host` on nxos7K and 9K, it seems that the hosts force the encrypt_type to 7 even providing 0 as encrypt_type, hence the idempotence issue. will you restrict some of these testcases to other versions of NXOS other than 7K and 9K? using `when: platform is search('N5K')`?

Or is there anyway to make the hosts not forcing the encrypt_type?